rego/Gmail.rego

```diff
@@ -1587,13 +1862,24 @@ if {
 #
 # Baseline GWS.GMAIL.15.1v0.1
 #--
```

Comment:
The output of this test is swapped: if I enable pre-delivery scanning, the test fails. If I disable it, the test passes.